src/extension.ts

```diff
@@ -13,6 +14,11 @@ import * as server from './server';
 // your extension is activated the very first time the command is executed
 export function activate(context: vscode.ExtensionContext) {
     let serversData;
+    const outputPanel = vscode.window.createOutputChannel('ssp-server');
```

Comment:
create one more channel for stderr stream and name channels:
* SSP Server (stdout)
* SSP Server (stderr)